docs/wintercredit/lexicon.rst

```diff
@@ -0,0 +1,84 @@
+possible from the official Hydro-Quebec electricy rates document (page
+31) :
+
+https://www.hydroquebec.com/data/documents-donnees/pdf/electricity-rates.pdf
+
+and from the “Régie de l’énergie” document :
+
+http://publicsde.regie-energie.qc.ca/projets/469/DocPrj/R-4057-2018-B-0062-DDR-RepDDR-2018_10_26.pdf#page=124
+
+**Period**
+    A period is time window where a specific billing or algorythmic logic is
+    applied by HQ
+
+        * normal
+            A period when nothing special is defined by HQ rate policies
+
+        * peak / peak hours
+            In this module:
+                All hours from 06:00 to 09:00 and from 16:00 to 20:00 during the winter.
+
+            This is when the critical peak events from hydro are happening.
+
+            In hydro’s document there are also exclusions for specific holiday dates (Christmans, New year, Good Friday
+            and Easter Monday) that we don’t take into account here (yet)
+
+        * anchor / temperature adjustment
+            This period starts 5 hours before the next peak event’s start time and has a duration of 3 hours. With the
+            current peak period (as described above) it results in the following time periods:
+
+                **Morning** 01h00-04h00
+
+                **Evening** 11h00-14h00
+
+            This period is used by HQ in combination with the reference period to calculate the Reference Energy used to
+            calculate the credit by trying to guess the additionnal energy usage caused by the colder temperature.
+
+            In HQ’s rate document it is called temperature adjustment and in the “Regie de l’énergie” docuement it is
+            refered to as an “anchor” period.
+
+        * reference period
+            This value is not used or provided in winter_module.py but is good
+            to know about to give some context to the calculations.
+
+            The period corresponding to the last 5 non-critical event, differentiated by weekend vs week days.
+
+            **Examples:**
+
+            Saturday evening critical peak event’s reference period = Last 5
+            non-critical evening peaks that occured on weekend days
+
+            Wednesday morning critical peak event’s reference period = Last 5
+            non-critical morning peaks that occurend on week days
+
+**event** | **critical peak event**
+    An event is also refered to a “critical peak event” means that HQ sent a
+    notification that the peak period will be considered critical and
+    admissible to winter credits
+
+**critical**
+    When a period is critical it can means two things depending on the period type:
+
+        **peak period**
+
+        The current peak period is part of a “critical peak event”
+
+        **normal period or anchor period**
+
+        Means that the next immediate peak period will be a “critical peak event”
+
+**none** | **normal** | **regular**
+    The period is not or will not be critical.
+
```

Comment:
I would remove critical peak event from here since it is a part of the description. Also the indentation seems off for this section.

The structure I had was the following where events can be of value critical or none with the description of each type.

- event
 - critical
 - none

Reply:
I was also hoping we could decide which value make more sense between none/normal/regular for the event status. I don't think we should keep all three.

Reply:
Actually, I did just that ! I picked normal, as none is a python keyword and is lacking some meaning, and normal was already in use.
Hope it works for you as well :)